cmd/insights/contributors.go

```diff
@@ -6,18 +6,18 @@ import (
 	"fmt"
 	"net/http"
 	"strconv"
+	"strings"
 	"sync"
 
-	"github.com/charmbracelet/bubbles/table"
-	"github.com/charmbracelet/lipgloss"
+	bubblesTable "github.com/charmbracelet/bubbles/table"
 	"github.com/open-sauced/go-api/client"
 	"github.com/open-sauced/pizza-cli/pkg/api"
 	"github.com/open-sauced/pizza-cli/pkg/constants"
 	"github.com/open-sauced/pizza-cli/pkg/utils"
 	"github.com/spf13/cobra"
 )
 
-type Options struct {
+type contributorsOptions struct {
```

Comment:
Is Go Pascal or camel cased when it comes to types?

Reply:
It's actually not a matter of style but a matter of syntax: if you want to export that type out of the package, it needs to start with a capital letter.

So `ContributorsOptions` would be exported out of the package while `contributorsOptions` would not be.

Reply:
TIL